tests/helpers.py

```diff
@@ -240,9 +267,14 @@ def create_pvc(sc_name, pvc_name=None):
     )
     pvc_data['metadata']['namespace'] = defaults.ROOK_CLUSTER_NAMESPACE
     pvc_data['spec']['storageClassName'] = sc_name
-    return create_resource(
-        desired_status=constants.STATUS_BOUND, **pvc_data
-    )
+    if size:
+        pvc_data['spec']['resources']['requests']['storage'] = size
+    ocs_obj = pvc.PVC(**pvc_data)
+    ocs_obj.create()
+    if wait:
+        ocs_obj.reload()
```

Comment:
maybe move the reload after the waiting for it?
Then you will get the actual status of the PVC

Reply:
please address this comment

Reply:
Yah, moved after wait_resource_state